.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches:
+      - master, github-action
+  pull_request:
+
+name: Gridiron
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
```

Comment:
Following the convention we set in `ironcore-events`, we should set this to 18.04

Reply:
I didn't realize we'd set that convention... is there a particular reason for it?

Reply:
@skeet70 it was first suggested here https://github.com/IronCoreLabs/ironcore-events/pull/59#discussion_r371542102, but we could still discuss it if you feel differently.

Reply:
The benefit of automatically upgrading, balanced against the cost of automatically upgrading. I prefer to have some control over this, so it doesn't break our builds when the default becomes 20.04 instead of 18.04. But as @giarc3 says, it could go either way.